docs/config.md

```diff
@@ -14,24 +14,41 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
+# Table of Contents
+
+- [Configuring Model Analyzer](#configuring-model-analyzer)
+- [Config options for **profile**](#config-options-for-profile)
+  - [CLI and YAML Config Options](#cli-and-yaml-config-options)
+  - [YAML Only Options](#yaml-only-options)
+- [Config Options for **report**](#config-options-for-report)
+  - [CLI and YAML options](#cli-and-yaml-options)
+  - [YAML only options](#yaml-only-options)
+- [Field Descriptions](#field-descriptions)
+- [Config Defaults](#config-defaults)
+
+<br>
+
 # Configuring Model Analyzer
 
-Each [subcommand](cli.md#model-analyzer-subcommands) in Model Analyzer can be configured with a
-[YAML](https://yaml.org/) config file or via the command line interface (CLI). Every
-flag supported by the CLI is supported in the configuration file, but some
-flags are only supported using the config file.
+Model Analyzer can be configured using either a [YAML](https://yaml.org/) config file, the `command line interface\* (CLI), or a combination of both.
+
+- **Every** flag supported by Model Analyzer can be configured using a **YAML** config file
+- Only a **subset** of flags can be configured using the **CLI**
 
-The placeholders below are used throughout the configuration:
+---
 
-- `<boolean>`: a boolean that can take `true` or `false` as value.
+The placeholders listed below are used throughout the configuration:
+
+- `<boolean>`: a boolean that can take `true` or `false` as value
 - `<string>`: a regular string
-- `<comma-delimited-list>`: a list of comma separated items.
-- `<int>`: a regular integer value.
-- `<list>`: a list of values.
+- `<comma-delimited-list>`: a list of comma separated items
+- `<int>`: a regular integer value
+- `<list>`: a list of values
 - `<range>`: An object containing `start` and `stop` key with an optional `step`
-  value. If `step` is not defined, we use 1 as the default value for `step`. The
-  types that support `<range>` can be described by a list or using the example
-  structure below:
+  value
+  - If `step` is not defined, **1** is the default value
```

Comment:
my first read of this implied range was 1 if step wasn't defined.  Can you reword it
``` If step is not defined it defaults to 1```

Reply:
Fixed